### ISSUE TITLE
Experimental "my" subs not enabled

### ISSUE THREAD

**Issue by eserte** [jimav/Text-Table-Boxed#1]
The test suite fails with perls <= 5.36 on my smoker systems. One failure pattern:
```
PERL_DL_NONLAZY=1 "/usr/perl5.22.2t/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
Experimental "my" subs not enabled at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/../lib/Text/Table/Boxed.pm line 90.
	require Text/Table/Boxed.pm called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12
	t_TTBUtils::BEGIN() called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/../lib/Text/Table/Boxed.pm line 90
	eval {...} called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/../lib/Text/Table/Boxed.pm line 90
	require t_TTBUtils.pm called at t/10_Basic.t line 8
	main::BEGIN() called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/../lib/Text/Table/Boxed.pm line 90
	eval {...} called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/../lib/Text/Table/Boxed.pm line 90
Compilation failed in require at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12.
	t_TTBUtils::BEGIN() called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12
	eval {...} called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12
	require t_TTBUtils.pm called at t/10_Basic.t line 8
	main::BEGIN() called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12
	eval {...} called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12.
	require t_TTBUtils.pm called at t/10_Basic.t line 8
	main::BEGIN() called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12
	eval {...} called at /home/cpansand/.cpan/build/2024121319/Text-Table-Boxed-1.002-oHMLT2/t/t_TTBUtils.pm line 12
Compilation failed in require at t/10_Basic.t line 8.
	main::BEGIN() called at t/10_Basic.t line 8
	eval {...} called at t/10_Basic.t line 8
BEGIN failed--compilation aborted at t/10_Basic.t line 8.
t/10_Basic.t ............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
... etc ...
```

**Comment by eserte** [comment]
Another one:
```
PERL_DL_NONLAZY=1 "/usr/perl5.28.3/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
The signatures feature is experimental at /home/cpansand/.cpan/build/2024121407/Text-Table-Boxed-1.002-1/t/t_TTBUtils.pm line 21.
Compilation failed in require at t/10_Basic.t line 8.
BEGIN failed--compilation aborted at t/10_Basic.t line 8.
t/10_Basic.t ............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

**Comment by jimav** [comment]
Ok, it should be fixed now with v1.003
Thanks for the report.
